README.md

```diff
@@ -1,18 +1,25 @@
-**R1	Description of your website, including:**
-- Purpose
-- Functionality / features
-- Target audience
-- Tech stack
+## R1.	Description of the app
+- **Description and purpose:** The app our team will build is a fitness tracking app for athletes and workout enthusiasts. The purpose of the app is to help users to log their fitness performance for a variety of exercises including weight lifting, running and cycling. This app can help users set their fitness goals and track their performance over time.
 
+- **Functionality / features:**
+    1. User signup. A user enters their username, email, password, age, weight, height, and optionally their profile picture when signing up.
+    2. User login. A user enter their email or username, and password to login.
+    3. User signout.
+    4. Select an exercise. A user can select their exercise that they want to keep track of.
+    5. Exercise log entry. After a user selects their exercise, they can enter in the specific details of their exercise (e.g. weight and reps).
+    6. History. A user can open up a calendar and go to any day in the past to view their exercise details.
+    7. Dashboard. A user can visualise their performance over time graphically.
```

Comment:
Perfect! I was thinking can we keep focus on first five features and later extend the scope to the 
1. history
2. . Dashboard
Because this can extend  our scope. What do you think?  But if you think we can complete the Dashboard too in time, then I am ok with it.